central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/manager_impl.go

```diff
@@ -589,7 +591,12 @@ func (m *managerImpl) getFixableDeferrals() ([]*storage.VulnerabilityRequest, er
 			if err != nil {
 				return nil, err
 			}
-			count, err := m.imageCVEs.Count(ctx, fixableQuery)
+			var count int
+			if features.FlattenCVEData.Enabled() {
+				count, err = m.imageCVE2.Count(ctx, fixableQuery)
```

Comment:
Make sure these counts return the proper counts.  For instance if the same CVE occurs multiple times in the same image, we probably only want to count that as 1.